product-updates/knowledge-center/viewing-error-reports.html

```diff
@@ -0,0 +1,695 @@
+      </td>
+<td>
+  Section 1 (Active Case Data)
+</td>
+    </tr>
+
+                <tr>
+      <th scope="row">T4, M4</th>
+      <td>
+        Family-level data for the last month the family received assistance. 
+      </td>
+<td>
+  Section 2 (Closed Case Data)
+</td>
+    </tr>
+
+                    <tr>
+      <th scope="row">T5, M5</th>
+      <td>
+        Person-level data for the last month that the family received assistance. 
+      </td>
+<td>
+  Section 2 (Closed Case Data)
+</td>
+    </tr>
+
+                        <tr>
+      <th scope="row">T6, M6</th>
+      <td>
+        Aggregate data of families applying for and receiving assistance, by month.
+      </td>
+<td>
+  Section 3 (Aggregate Data)
+</td>
+    </tr>
+
+                            <tr>
+      <th scope="row">T7, M7</th>
+      <td>
+        Aggregate data of families  receiving assistance, by month and stratum.
+      </td>
+<td>
+  Section 4 (Stratum Data)
+</td>
+    </tr>
+      
+  </tbody>
+</table>
+           <div style="min-height: 22px;"></div>
+               <h4>Header and Trailer Records</h4>
+                <div style="min-height: 22px;"></div>
+                <p>The <a href="https://www.acf.hhs.gov/sites/default/files/documents/ofa/transmission_file_header_trailer_record.pdf">Header and Trailer</a> refer to special records at the beginning and end of every data file. The Header communicates key information to TDP about the file's classification that helps the system correctly process it, including calendar year and quarter, program type, and section. The Trailer contains information about the number of records (excluding the header and trailer records) in the file. </p>
+                  <div style="min-height: 22px;"></div>
+     <p>Examples of Header and Trailer records:</p>
+         
+
+
+<blockquote style="margin-left: 0px; padding-left: 10px; padding-bottom: 10px; border-color: blue; border-left: solid 3px #005ea2; font-size: 17px; font-weight: 500; line-height: 20px; background-color: #ebf9fc; border-radius: 10px;"><span style="font-size: 40px; color: #005ea2; position: relative; right: 10px; padding-left: 10px;">“</span><br>HEADER20204G02000TAN2 D <br>
+    TRAILER0000042 
+          <span style="font-size: 40px; position: relative; top: 37px; left: 10px; color: #005ea2">”</span></blockquote>
+
+           <p>Typically, these records will be generated automatically as part of your current process for exporting data files. Please note: TDP does require that the update indicator, one of the items making up the Header record, be set to a value of "D" (meaning "Delete"). The update indicator is an instruction to the system to delete and replace any existing data in the TDP database for the specified quarter, program type, and section with the data included in the submitted file). <a href="/exporting-complete-data-using-ftanf">Read more about how to submit complete data.</a></p>
```

Comment:
im not sure if the included link is the correct one, since not all STTs use FTANF. maybe this one: https://tdp-project-updates.app.cloud.gov/knowledge-center/complete-resubmissions.html

It's also not a perfect solution since the page is focused on resubmissions.